Explicit Browser Support (#1682)
chanzuckerberg/cellxgene: README.md

- [Preparing your own data](https://chanzuckerberg.github.io/cellxgene/posts/prepare) for use in cellxgene
- Checking out [our roadmap](https://chanzuckerberg.github.io/cellxgene/posts/roadmap) for future development
- [Contributing](https://chanzuckerberg.github.io/cellxgene/posts/contribute) to cellxgene

### Quick start

To install cellxgene you need Python 3.6+. We recommend [installing cellxgene into a conda or virtual environment.](https://chanzuckerberg.github.io/cellxgene/posts/install)

Install the package.

```bash
pip install cellxgene
```

Launch cellxgene with an example [anndata](https://anndata.readthedocs.io/en/latest/) file

```bash
cellxgene launch https://cellxgene-example-data.czi.technology/pbmc3k.h5ad
```

To explore more datasets already formatted for cellxgene, check out the [Demo data](https://chanzuckerberg.github.io/cellxgene/posts/demo-data) or
see [Preparing your data](https://chanzuckerberg.github.io/cellxgene/posts/prepare) to learn more about formatting your own
data for cellxgene.

### Supported browsers

cellxgene currently supports the following browsers:

- Google Chrome 61+
- Edge 15+
- Firefox 60+
- Safari 10.1+

> Please [file an issue](https://github.com/chanzuckerberg/cellxgene/issues/new/choose) if you find that your browser is not supported or if would like us to expand our support to other browsers.

Reviewer: ```suggestion
Please [file an issue](https://github.com/chanzuckerberg/cellxgene/issues/new/choose) if you would like us to add support for an unsupported browser.
```

A bit more concise.

> 

Author: There are a number of browsers that we don't yet know if we support (eg. Opera, QQ, Navigator)

> 

Author: Do the readme additions need to be reflected somewhere on the cellxgene site? If so, where?

Author: cc: @sidneymbell
